pkg/webhook/controlplane/utils.go

```diff
@@ -18,9 +18,33 @@ import (
 	"reflect"
 	"strings"
 
+	"github.com/coreos/go-systemd/unit"
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
 	corev1 "k8s.io/api/core/v1"
 )
 
+var cmdLineSpace = [256]uint8{'\t': 1, '\n': 1, '\v': 1, '\f': 1, '\r': 1, ' ': 1, '\\': 1}
+
+// DeserializeCommandLine deserializes the given string to a slice of command line elements by splitting it
+// on white space and the "\" character.
+func DeserializeCommandLine(s string) []string {
```

Comment:
Why not split by a regex? For whitespace you could use `\s`, which makes the `var` above unnecessary.

Reply:
Good idea, done.